linode_api4/objects/object_storage.py

```diff
@@ -1,4 +1,155 @@
+    ):
+        """
+        Allows changing basic Cross-origin Resource Sharing (CORS) and Access Control
+        Level (ACL) settings. Only allows enabling/disabling CORS for all origins,
+        and/or setting canned ACLs. For more fine-grained control of both systems,
+        please use the more fully-featured S3 API directly.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-bucket-access-modify
+
+        :param cluster_id: The ID of the cluster this bucket exists in.
+        :type cluster_id: str
+
+        :param bucket: The bucket name.
+        :type bucket: str
+
+        :param acl: The Access Control Level of the bucket using a canned ACL string.
+                    For more fine-grained control of ACLs, use the S3 API directly.
+        :type acl: str
+                   Enum: private,public-read,authenticated-read,public-read-write
+
+        :param cors_enabled: If true, the bucket will be created with CORS enabled for
+                             all origins. For more fine-grained controls of CORS, use
+                             the S3 API directly.
+        :type cors_enabled: boolean
+        """
+        params = {
+            "acl": acl,
+            "cors_enabled": cors_enabled,
+        }
+
+        resp = self._client.post(
+            "/object-storage/buckets/{}/{}/access".format(cluster_id, bucket),
+            data=drop_null_keys(params),
+        )
+
+        if "errors" in resp:
+            raise UnexpectedResponseError(
+                "Unexpected response when modifying the access to a bucket!",
+                json=resp,
+            )
+        return True
+
+    def access_update(
+        self,
+        cluster_id,
+        bucket,
+        acl: ObjectStorageACL = None,
+        cors_enabled=None,
+    ):
+        """
+        Allows changing basic Cross-origin Resource Sharing (CORS) and Access Control
+        Level (ACL) settings. Only allows enabling/disabling CORS for all origins,
+        and/or setting canned ACLs. For more fine-grained control of both systems,
+        please use the more fully-featured S3 API directly.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-bucket-access-update
+
+        :param cluster_id: The ID of the cluster this bucket exists in.
+        :type cluster_id: str
+
+        :param bucket: The bucket name.
+        :type bucket: str
+
+        :param acl: The Access Control Level of the bucket using a canned ACL string.
+                    For more fine-grained control of ACLs, use the S3 API directly.
+        :type acl: str
+                   Enum: private,public-read,authenticated-read,public-read-write
+
+        :param cors_enabled: If true, the bucket will be created with CORS enabled for
+                             all origins. For more fine-grained controls of CORS,
+                             use the S3 API directly.
+        :type cors_enabled: boolean
```

Comment:
The boolean type in Python is defined as `bool`, instead of `boolean`

Reply:
The comment is just for understanding, I don't feel it has to be strictly typed.

Reply:
But you can see the string was as `str`

Reply:
Some other functions also apply Python tying, so I think it would be nice to keep it consistent.

For example:

https://linode-api4.readthedocs.io/en/latest/linode_api4/objects/models.html#linode_api4.objects.database.MongoDBDatabaseBackup.save